packages/app-tests/e2e/document-flow/stepper-component.spec.ts

```diff
@@ -333,3 +491,45 @@ test('[DOCUMENT_FLOW]: should be able to create, send with redirect url, sign a
 
   await unseedUser(user.id);
 });
+
+test('[DOCUMENT_FLOW]: should be able to sign a document with custom date', async ({ page }) => {
+  const user = await seedUser();
+  const customDate = DateTime.local().toFormat('yyyy-MM-dd hh:mm a');
+
+  const { document, recipients } = await seedPendingDocumentWithFullFields({
+    owner: user,
+    recipients: ['user1@example.com'],
+    fields: [FieldType.DATE],
+  });
+
+  const { token } = recipients[0];
+
+  await page.goto(`/sign/${token}`);
+  await page.waitForURL(`/sign/${token}`);
+
+  await page.getByTestId('field').click();
```

Comment:
Don't hardcode the test id since it'll cause issues when there are more than one

Do something like this instead

```
      await page.locator(`#field-${field.id}`).getByRole('button').click();
```

Example taken from this test

```
[DOCUMENT_AUTH]: should deny signing fields when required for global auth
```